cmd/server/app/run_service.go

```diff
@@ -241,6 +244,11 @@ func (h *serviceHandler) Execute(_ []string, r <-chan svc.ChangeRequest, s chan<
 		h.errC <- h.srv.Serve(ctx)
 	}()
 
+	// start system agent
+	go func() {
+		h.errC <- h.agent.Run(ctx)
```

Comment:
Should we be mixing the use of this channel wtih agent and server?

Reply:
That channel is for the windows service, so both "sub processes" should be reporting into that channel.